package.json

```diff
@@ -21,7 +21,7 @@
     "truffle": "truffle",
     "build": "rimraf build && truffle compile",
     "docs:gen": "node docs/scripts/gen.js",
-    "publish": "npm run docs:gen"
+    "publish": "truffle build && npm run docs:gen && network --clean && npm publish"
```

Comment:
> "truffle build && npm run docs:gen && network --clean && npm publish"

Did you tested this syntax ?
It should be more like ...
> "truffle compile && npm run docs:gen && truffle network --clean && npm publish"

Reply:
Well, I'm not really familiar with how we should deploy Arc. 
As it's not really relevant to this PR, I think we should add a publish script on a different PR and then add `docs:gen`. 
Agreed?

Reply:
ok. please remove the publish.
I will add that later.